packages/dotnet/ReadmeMetricsAPI5/Controllers/StudentController.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ReadmeMetricsAPI5.Controllers
```

Comment:
@Dashron What's this `ReadmeMetricsAPI3` and `ReadmeMetricsAPI5` and `ReadmeMetricsAPI6` stuff?

Reply:
They are test apps written in .net versions 3, 5 and 6 respectively. Still a little up in the air what to do with these things 🤷‍♂️ but leaving them there for now. I can imagine us creating a standard of example apps per language in an `examples/` folder in each subpackage, but i'm okay with these here for now.